google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneGcpDialog.java

```diff
@@ -52,22 +54,22 @@
  * The dialog that prompts the user to download (git clone) from a GCP project.
  */
 public class CloneGcpDialog extends DialogWrapper {
+
   private static final Logger LOG = Logger.getInstance(CloneGcpDialog.class);
   public static final String INVALID_FILENAME_CHARS = "[/\\\\?%*:|\"<>]";
```

Comment:
This variable doesn't seem to be used anywhere else other than this class.
LOG is also showing as unused